src/poliastro/tests/tests_plotting/test_porkchop.py

```diff
@@ -0,0 +1,21 @@
+import matplotlib.pyplot as plt
+import pytest
+
+from poliastro.bodies import Earth, Mars
+from poliastro.plotting.porkchop import porkchop
+from poliastro.util import time_range
+
+
+@pytest.mark.mpl_image_compare
+def test_porkchop_plotting():
+
```

Comment:
Remove this extra line?

Reply:
Oh, so sorry! Forgot to clean this before uploading it :sweat_smile: